M/manifold/bundled/patches/submodules.patch

```diff
@@ -0,0 +1,14 @@
+diff --git before/manifold/.git/config after/manifold/.git/config
```

Comment:
You don't seem to be applying this patch (or including it). What is this for?

Reply:
I am not sure, the wizard created it.

Reply:
Huh, I haven't heard of the wizard making diffs like this before, @giordano is this something it actually does?

Either way, it looks like it isn't needed since it isn't being applied directly, so maybe we can remove it and not need the bundled directory anymore?

Reply:
Yes, it keeps a copy of the original filesystem before the user starts the build process, then it makes a diff with the filesystem at the end and creates the patches. Probably files under a .git directory should be automatically excluded though.